faq/index.md

```diff
@@ -2,153 +2,280 @@
 
-## <a name="node"></a>What is a node?
+## <a name="ownership"></a>Who owns NYC Mesh?
 
-A node is a location with one or more routers or antennas, i.e. it is usually a dot on our map. Some dots have multiple apartments with installs in the same building so that would be one node per apartment. A supernode will have multiple radios, routers, antennas and maybe a server too.
+We are a non-profit project of the New York chapter of the [Internet Society](https://www.internetsociety.org/) (ISOC-NY). Most of the equipment that makes up the Mesh is owned by the individual members. Some common infrastructure is owned or rented by the project under ISOC-NY, or by individual members who are donating it. Intellectual property is limited and is under open licenses such as GNU and Creative Commons. 
 
-## <a name="router"></a>What is the difference between a router and an antenna?
+## <a name="businessplan"></a>What is your business plan?
+We are not a business and we are not a commercial Internet Service Provider! We aim to grow our community in a sustainable and inclusive way for the benefit of all of our members. We don’t see members as customers, consumers or users, but as contributors to our community. Being a member of our network has many benefits, including replacing your current ISP with fast, neutral, affordable, and unmonitored Internet access. Businesses are free to join or connect to the mesh as long as they obey the [network commons license](../ncl.pdf).
 
-Most of the devices we use, such as a LiteBeam or NanoStation are self-contained so they have an antenna, radio and ethernet router all in one. "Wireless router with antenna" is too long so we usually call the whole thing a router, just like your home wifi "router". Sector antennas usually have a separate radio plugged into the back of them so we call those antennas (although some sectors are self-contained). 
+## <a name="meshonly"></a>What can I do on the mesh that I can’t do on the internet?
 
-Ubiquiti call their self-contained AirFiber a "radio", and others call an AirFiber a router or an antenna, so yes everyone is confused about what term to use.
+Other than the benefit of not paying the cable cartel for overpriced Internet access, it is also possible to connect directly to other people on the mesh without using the Internet. If for some reason the Internet is shut down (hurricane, government crackdown, cyberattack, zombie apocalypse), the mesh will likely still work for a while longer. 
 
-## <a name="firmware"></a>What software/firmware do you use?
+The mesh also has its own web servers that show local information and allow you to chat and post messages. So far we have servers in five locations: Harlem, Bay Ridge, 3rd St. and 7th St. in the East Village, and our main directory “wiki.mesh” in Williamsburg. We are in need of [volunteers](../help) to help add content and maintain our servers.
```

Comment:
Does wiki.mesh actually work?

Reply:
It may not work today, but it is a valid service that we should get working. Similarly, some services may not be available in neighborhoods, but it doenst mean they don't exist. ( wiki could be considered one of those ). It might be good to give examples of services that can exist rather than direct links.